bubble-up-changes.sh

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env bash
+# Like this script? 
+# Generate boilerplate for similar ones at https://bashplate.wolfgang-werner.net.
```

Comment:
:heart_eyes:

Reply:
Will remove the shameless plug, though - it's in the template, forgot to remove :-)

Reply:
Noooooo! Otherwise I would have never learned about it!